be/src/cloud/cloud_warm_up_manager.cpp

```diff
@@ -147,15 +147,13 @@ void CloudWarmUpManager::handle_jobs() {
                     auto schema_ptr = rs->tablet_schema();
                     auto idx_version = schema_ptr->get_inverted_index_storage_format();
                     if (idx_version == InvertedIndexStorageFormatPB::V1) {
-                        for (const auto& index : schema_ptr->indexes()) {
-                            if (index.index_type() == IndexType::INVERTED) {
```

Comment:
why remove the check

Reply:
The new interface `schema_ptr->inverted_indexes()` has already filtered the inverted index types.